hermes-main-website/src/app/components/footer/footer.component.sass

```diff
@@ -0,0 +1,124 @@
+
+@import "src/variables.sass"
+
+
+
+.container
+  width: 100%
+  position: absolute
+  bottom: 0
+  color: $white-main
```

Comment:
don't think you need this color here, but If it applies to color to all sub-elements, keep it here and delete it from the other parts text-align-center etc